tools/hooks/30-json.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+echo "Executing 'json' formatter..."
+
+TARGET_FILES=`$TRACKED_FILES | grep -P '\.json'`
+
+pushd $PROJECT_ROOT > /dev/null
+    for FILE in ${TARGET_FILES}; do
+        python -mjson.tool $FILE 2>&1 /dev/null
```

Comment:
Nit: use `python -m json.tool`, as its more readable. Also, the tool needs `--sort-keys` set here and below.